### ISSUE TITLE
 AWS::Serverless::HttpApi and AWS::Serverless::Api both have a Description property

### ISSUE THREAD

**Issue by kstro21** [aws-cloudformation/cfn-python-lint#1908]
*cfn-lint version: (` 0.45.0`)*

Resource of type [AWS::Serverless::HttpApi](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-httpapi.html) and [AWS::Serverless::Api](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-api.html) both have a `Description` property in the docs but if I add it to a template then `cfn-lint` fails with 
>[cfn-lint] E0001: Error transforming template: Resource with id [MyApi] is invalid. property Description not defined for resource of type AWS::Serverless::HttpApi

Here is an example

```YAML
MyApi:
    Type: 'AWS::Serverless::HttpApi'
    Properties:
      Description: 'Client API'
      StageName: !Ref 'StageName'
      CorsConfiguration:
        AllowCredentials: true
        AllowMethods: "'GET,POST,OPTIONS'"
        AllowHeaders: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'"
        AllowOrigins: '*'
```

The template deploys ok using `sam deploy` so it is not an error. I'm already using the latest version of `cfn-lint` and already did `cfn-lint -u`

Hope this gets fixed soon.

PS: nice job!

**Comment by PatMyron** [comment]
Unable to reproduce with the latest `aws-sam-translator` version
could you check `pip3 show aws-sam-translator | grep 'Version'`?
and see if it still occurs after `pip3 install aws-sam-translator --upgrade`?

---

Another potential issue might just be the unmatched quote in `Description: Client API'`

**Comment by kstro21** [comment]
Awesome! Thanks, @PatMyron `pip3 install aws-sam-translator --upgrade` worked. I had `aws-sam-translator 1.25.0` and now I have `aws-sam-translator 1.34.0`

I can't find `pip3 install aws-sam-translator --upgrade` in any doc and invoking `pip3 install cfn-lint --upgrade` doesn't seems to also upgrade `aws-sam-translator`. Is that ok or should `aws-sam-translator` be upgraded too?

**Comment by PatMyron** [comment]
> invoking `pip3 install cfn-lint --upgrade` doesn't seems to also upgrade `aws-sam-translator`

Wish we could pin the latest version without needing code changes:
https://github.com/aws-cloudformation/cfn-python-lint/blob/002b3d1fd559698fad5c31057486353598210951/setup.py#L48
Could at least set up [Dependabot](https://dependabot.com/) to automate that
